packages/babel-preset-env/test/debug-fixtures/entry-corejs3-versions-strings-minor-3.1/stdout.txt

```diff
@@ -161,6 +162,7 @@ Using polyfills with `entry` option:
   es.string.includes { "ie":"10" }
   es.string.iterator { "ie":"10" }
   es.string.match { "ie":"10" }
+  es.string.match-all { "chrome":"54", "ie":"10", "node":"6.10" }
   es.string.pad-end { "chrome":"54", "ie":"10", "node":"6.10" }
   es.string.pad-start { "chrome":"54", "ie":"10", "node":"6.10" }
```

Comment:
This is the only fixture that actually changes something. All the other ones just moved the import from a to b.